core/src/KokkosExp_MDRangePolicy.hpp

```diff
@@ -30,6 +30,7 @@ static_assert(false,
 #include <impl/KokkosExp_Host_IterateTile.hpp>
 #include <Kokkos_ExecPolicy.hpp>
 #include <type_traits>
+#include <sstream>
```

Comment:
can we avoid pulling the sstream header into this and just use a regular string? (see my comment below)

Reply:
Removed in [341b464](https://github.com/kokkos/kokkos/pull/6617/commits/341b464d38638c07fb4767c4becb35ab78859dbd).